dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -747,14 +749,15 @@ private void cacheDefaultExtensionName() {
 
         String value = defaultAnnotation.value();
         if ((value = value.trim()).length() > 0) {
-            String[] names = NAME_SEPARATOR.split(value);
+            cachedDefaultName = value;
+           /* String[] names = NAME_SEPARATOR.split(value);
             if (names.length > 1) {
                 throw new IllegalStateException("More than 1 default extension name on extension " + type.getName()
                         + ": " + Arrays.toString(names));
             }
             if (names.length == 1) {
                 cachedDefaultName = names[0];
-            }
+            }*/

```

Comment:
Why comment this snippet?

Reply:
If there are multiple type names separated by commas, an error will be directly reported below, that is, multiple types cannot be displayed, so why do we need to use separators? Think of it as a type name